kots-preflight.yaml

```diff
@@ -63,3 +63,27 @@ spec:
               message: The cluster should contain at least 4 cores.
           - pass:
               message: There are at least 4 cores in the cluster.
+        checkName: Required Node Available Disk Space
+        #The below outcomes are intentionally set to very high values as these should be adjusted based on the needs of your application.
+        strict : true
+        #The above setting prevents this preflight check from allowing a user to deploy if the outcome is `false`
+        outcomes:
+          - fail:
+              when: "min(ephemeralStorageCapacity) < 5120Gi"
```

Comment:
But I like the idea. In the past we had something with the container runtime checking for `docker`. Which sometimes failed and was a good trigger to fix/update the preflights.

Reply:
I should stop going through these in order because this comment may have invalidated my above response to your earlier comment :D